src/main/java/teammates/ui/controller/InstructorFeedbackEditPageAction.java

```diff
@@ -25,8 +25,11 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         Assumption.assertPostParamNotNull(Const.ParamsNames.COURSE_ID, courseId);
         String feedbackSessionName = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
         Assumption.assertPostParamNotNull(Const.ParamsNames.FEEDBACK_SESSION_NAME, feedbackSessionName);
-        String loadInEditMode = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_ENABLE_EDIT);
-        Assumption.assertPostParamNotNull(Const.ParamsNames.FEEDBACK_SESSION_ENABLE_EDIT, loadInEditMode);
+        String loadInEditModeParam = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_ENABLE_EDIT);
+        boolean loadInEditMode = false;
+        if (loadInEditModeParam != null && "true".equals(loadInEditModeParam)) {
```

Comment:
Writing `"true".equals(loadInEditModeParam)` already takes care of the null case. The whole thing can be simplified to
```
boolean loadInEditMode = "true".equals(loadInEditModeParam);
```